newrelic_rpm.gemspec

```diff
@@ -61,4 +61,8 @@ https://github.com/newrelic/newrelic-ruby-agent/
   s.add_development_dependency 'bundler'
   s.add_development_dependency 'rubocop'
   s.add_development_dependency 'rubocop-performance'
+  if RUBY_VERSION >= '2.7.0'
```

Comment:
Ooh, if we have this restriction, we could probably remove the conditional from [.simplecov](https://github.com/newrelic/newrelic-ruby-agent/pull/1171/files#diff-24002d905f0809599eae02b5cd7b58f8821aa89272b2b207c3e589b7a7a4f3a6) altogether.